ssp

```diff
@@ -1027,16 +1028,19 @@ sub get_os_info {
         chomp $line;
         $line =~ s/^\s+|\s+$//;
         if    ( length $line >= 4 )                                             { $release = $line; }
-        if    ( $line =~ m/(?:Corporate|Advanced\sServer|Enterprise|Amazon)/i ) { $ises    = 1; }
-        elsif ( $line =~ /CloudLinux|CentOS/i )                                 { $ises    = 2; }
+        if    ( $line =~ m/(?:Corporate|Advanced\sServer|Enterprise|Amazon)/i ) { $is_es   = 1; }
+        elsif ( $line =~ /CloudLinux|CentOS/i )                                 { $is_es   = 2; }
 
         if    ( $line =~ /(\d+\.\d+)/ ) { $version = $1; }
         elsif ( $line =~ /(\d+)/ )      { $version = $1; }
 
         if ( $line =~ /(centos|cloudlinux|amazon)/i ) { $os = lc $1; }
     }
 
-    return ( $release, $os, $version, $ises );
+    $is_amazon     = 1 if $os eq "amazon";
```

Comment:
```
$os eq 'amazon' ? $is_amazon = 1 : $is_amazon = 0;
```

We could really set that flag in a more intelligent way to reduce lines of code somewhere and keep a general global instead of multiple smaller ones.

In the least we don't need double quotes here for the comparison - it's better to use single quotes when working with string comparisions & eq / ne / lt / gt